### PR TITLE
[unitest] polish zero config in unittest

### PR DESCRIPTION
**Pull request by feifeibear**

### CODE REVIEW

File: tests/test_zero_data_parallel/common.py [hunk 1]
```diff
@@ -10,6 +10,18 @@
 
 LOGGER = get_dist_logger()
 
+_ZERO_OPTIMIZER_CONFIG = dict(optimizer_type=torch.optim.Adam, optimizer_config=dict(lr=1e-3))
```
Comment (FrankLeeeee) [inline tests/test_zero_data_parallel/common.py:13]: Do we need to follow the previous schema? The conventional schema we used previous is like:

```python
dict(
    type="Adam",
    lr=1e-3
)
```

The optimizer is registered in a registry and the build function will build the object via this schema

Comment (feifeibear) [reply tests/test_zero_data_parallel/common.py:13]: OK, I will improve it in another MR. I think a reactor implementation is needed to map the name to optimzer_cls.

Comment (feifeibear) [reply tests/test_zero_data_parallel/common.py:13]: I see. you mean I can use the global registry OPTIMIZERS. I will fix it in another MR ASAP.

Comment (FrankLeeeee) [reply tests/test_zero_data_parallel/common.py:13]: OK.

Comment (FrankLeeeee) [reply tests/test_zero_data_parallel/common.py:13]: Some registries are deprecated, I will remove them in another MR later.

**Review by FrankLeeeee — APPROVED** [review]

**Pull request merged**